src/collections/operations/restoreVersion.ts

```diff
@@ -97,7 +92,7 @@ async function restoreVersion<T extends TypeWithID = any>(args: Arguments): Prom
   const prevDocWithLocales = await getLatestCollectionVersion({
     payload,
     id: parentDocID,
-    query: findArgs,
+    query: findOneArgs,
```

Comment:
I believe this should be `findOneArgs.where`, right?

Reply:
@jmikrut I modified the `getLatestCollectionVersion` function so that it requires all the arguments instead of just the `where` - so in this case, findOneArgs should be correct.

You can find my changes to the getLatestCollectionVersion in [this PR](https://github.com/payloadcms/payload/pull/2880/files#diff-611658394554df67d53853bf8ce2e212886079c22789f0911ebcee1749853ede).